src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
 public class Application {
+
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+
+        Lottery lottery = new Lottery();
+        Ball [] balls = new Ball[NUMBER_OF_BALLS];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+            System.out.println(balls[i]);
+        }
```

Comment:
```suggestion
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall());
        }
```

It is not necessary to create an array. You can simply immediately display the resulting object, without an array.